blue-team/linux/change-passwords.sh

```diff
@@ -17,13 +17,15 @@ if [[ -z "$passTemp" ]] || [[ "$passTemp" != "$passConfirm" ]]; then
 	exit 2
 fi
 
-# host=$(hostname)
-
 for user in $(awk -F: '$7 ~ /(\/sh|\/bash)/ { print $1 }' /etc/passwd); do
 	if [[ $(id -u $user) -eq 0 ]]; then # Don't change password for root
 		continue
 	fi
+  touch ./users.csv
 	echo "Changing password for $user"
 	newPass="$passTemp$delim$user"
 	echo -e "$newPass\\n$newPass" | passwd "$user"
```

Comment:
Can use `chpasswd` to make it slightly cleaner
`echo "$user:$newPass" | chpasswd`

Reply:
Gotcha, commented old line (in case a machine is missing `chpasswd` for some reason) and added the line you suggested